CHANGELOG.md

```diff
@@ -14,7 +14,8 @@
 
 #### Bug Fixes
 
-* None.
+* Fix parsing of SwiftLint commands containing a URL in their trailing comment.  
```

Comment:
can you please rebase and fix this conflict? Thanks!

Reply:
I merged `master` and resolved this conflict.